Guide/src/reference/openvmm/management.md

```diff
@@ -0,0 +1,7 @@
+# Configuration and Management
+As [acknowledged in the disclaimer](../../user_guide/openvmm.md#notable-features) management interfaces in OpenVMM are unstable and incomplete. You can do basic VM operations such as stop, restart, save,  restore, pause, resume, etc.. via the interactive console. You can deploy a VM with various configurations such as number of processors, RAM size, UEFI, graphic console, etc.. and devices such as virtual NIC, vTPM, etc.. via the CLI. gRPC / ttrpc support functionality similar to the interactive console.
+
+## Missing Functionality
+  * List all the VMs 
```

Comment:
I don't think we're planning to support multi-vm workloads on a single process any time soon, right? Should we remove this bullet point?

Reply:
One of the goals of these ttrpc/grpc interfaces is to allow such tools to exist and interact with the openvmm instances they're managing.

Reply:
That's not exactly correct.

Something that might not be obvious from the current open-source code is that these interfaces were written 3+ years ago as a POC, and were left mostly untouched until quite recently, when grpc was added as a transport option. These interfaces - much like the CLI interfaces - are not representitive of any cohesive long-term "strategy" around "productizing" OpenVMM's management interfaces.

Steven's comment wrt. using a separate tool to manage multiple OpenVMM instances is just one choice amongst many that we could make wrt. how we want to build out our management stack. Other approaches - such as having a single OpenVMM instance manage multiple partitions - is also a viable design. Which one we go with is unclear at the moment, as we haven't yet dedicated proper deisgn time to OpenVMM's host-vmm management stack.

What that means right now is that we don't really have good answers to the questions you're asking @sriharshamadala. Like steven alluded to - we don't have plans to invest here in the near future, but I strongly suspect that sometime in the medium-term future, we will begin to invest more seriously into OpenVMM's host-side management interfaces.